R/convertToSE.R

```diff
@@ -0,0 +1,40 @@
+#' It helps to store any matrix in \linkS4class{SummarizedExperiment}
+#' object class data container(`SE`). The resulting abundance matrix from the
+#' simulation functions used in `miaSim` can be used.
+#'
+#' @param matrix : A matrix-like or list of matrix-like objects.
+#' Rows: genes, genomic coordinates, etc. Columns: samples, cells, etc.
+#' @param tse : Logical, decides if the object is stored in
+#' \linkS4class{TreeSummarizedExperiment} or \linkS4class{SummarizedExperiment}
+#' @param ... : additional objects that can be implemented in
+#' the `SE` object.
+#'
+#' @examples
+#' ExampleHubbellRates <- miaSim::simulateHubbellRates(
+#'     community_initial = c(0,5,10), migration_p = 0.01,
+#'     metacommunity_p = NULL, k_events = 1, growth_rates = NULL, norm = FALSE,
+#'     t.end=1000)
+#'
+#' HubbellSE <- convertToSE(matrix = ExampleHubbellRates$abundancematrix,
```

Comment:
Thinking of consistency: could we replace "abundancematrix" element name in `simulateHubbellRates` with just "counts", so we could refer to "ExampleHubbellRates$counts" instead of "ExampleHubbellRates$abundancematrix".

Our SE framework is using the name "counts" rather systematically, it would be conceptually more clear for compatibility purposes to use the same names for the analogous elements. 

And I would add the same comment for all simulation functions that have "abundancematrix" as output element.